app/src/main/java/org/simple/clinic/bp/BloodPressureMeasurement.kt

```diff
@@ -14,7 +14,7 @@ import org.threeten.bp.Instant
 import java.util.UUID
```

Comment:
Aren't we adding tests for the rest of the repositories to verify the pending records count?

Reply:
I remember discussing this with you and we decided its not required since its just a count.

Reply:
That said, I think we _eventually_ should. It's possible to write an incorrect SQL query.